src/components/ContextMenuView/ChatContextMenuView.tsx

```diff
@@ -21,6 +22,7 @@ type ChatContextMenuViewProps = {
   children: React.ReactNode;
   contextMenuType: BetterSocialContextMenuType;
   chat?: ChatSchema;
+  type: Anonimity;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding the `type: Anonimity;` property to the `ChatContextMenuViewProps` interface is a good practice as it ensures that the `ChatContextMenuView` component receives the correct type of chat. This will help prevent potential bugs and improve code readability.
```diff
+   type: Anonimity;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->